test/pseudo-tty/test-stderr-stdout-handle-sigwinch.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const common = require('../common');
+
+// if printing, add to out file
+const originalRefreshSizeWrapperStderr = process.stderr._refreshSize;
+const originalRefreshSizeWrapperStdout = process.stdout._refreshSize;
+
+const refreshSizeWrapperStderr = () => {
+  console.log('calling stderr._refreshSize');
+  try {
+    originalRefreshSizeWrapperStderr.call(process.stderr);
+  } catch (e) {
+    // EINVAL happens on SmartOS if emulation is incomplete
+    if (!common.isSunOS || e.code !== 'EINVAL')
+      throw e;
+  }
+};
+
+const refreshSizeWrapperStdout = () => {
+  console.log('calling stdout._refreshSize');
```

Comment:
A comment indicating that the `console.log` is part of the test would be helpful.